### ISSUE TITLE
[FEATURE REQUEST] Allow camel casing to be turned off

### ISSUE THREAD

**Issue by gkadillak** [yusinto/ld-redux#39]
Hello! First of all, loving the library. It fills a real need that we've had in our codebase and the source for it is dead simple. However, when refactoring code to use this library, renaming each flag from dashes to camel case was burdensome and error-prone. It would be great if there was a way to keep the flags as dash-cased on init. One idea I had was to add another option in the `init` method:

```js
ldRedux.init({
  clientSideId: 'your-client-side-id',
  camelCase: false,
  dispatch: store.dispatch,
  flags,
});
```

I'd be more than happy to put up a pr for this feature myself. What are your thoughts on this approach?

**Comment by yusinto** [comment]
Sounds good! Please put up a pr if you can and I'll take a look. Thanks for submitting this.

**Comment by gkadillak** [comment]
pr here @yusinto: https://github.com/yusinto/ld-redux/pull/40

**Comment by gkadillak** [comment]
@yusinto have you taken a look at this pr yet?

**Comment by yusinto** [comment]
This was merged in #40 . Closing now.
